config___settings.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+// check for a valid redaxo backend admin user session.
+$baseUrl = 'https://%%HOST%%'.$_SERVER['REQUEST_URI'];
```

Comment:
aus sicherheitsgründen wird die base-url für den single-sign-on bei installation des addons fest hier in die config geschrieben.

d.h. wenn man lokal entwickelt und dann die installation woanders hinkopiert muss man das addon re-installieren damit die sso-baseurl für die neue url der installation dann passt

Reply:
Super Idee